external-api/src/v1/controllers/estore/eStoreApi.ts

```diff
@@ -105,13 +107,8 @@ export const createFacilityFolder = async (siteId: string, dealIdentifier: strin
   return response;
 };
 
-export const uploadSupportingDocuments = async (
-  siteId: string,
-  dealIdentifier: string,
-  buyerName: string,
-  file: EstoreDealFiles,
-): Promise<UploadDocumentsResponse> => {
+export const uploadSupportingDocuments = async (siteId: string, dealIdentifier: string, file: EstoreDealFiles): Promise<UploadDocumentsResponse> => {
```

Comment:
We should add `buyerName: string` to the `EstoreDealFiles` interface, it's okay if that's a separate PR though

Reply:
updated now